server/app/services/PageNumberBasedPaginationSpec.java

```diff
@@ -0,0 +1,35 @@
+package services;
+
+import java.util.Optional;
+
+/**
+ * Specifies pagination behavior for a query with page number-based pagination. With page-number
+ * based pagination, the current page of results is resolved by multiplying the page size by the
+ * current page - 1 to derive an offset for the the underlying query.
```

Comment:
Document in the class more that page is 1 based, not 0.

Reply:
done and added to the getter